testutils/certs/README.md

```diff
@@ -0,0 +1,54 @@
+# Test Certificates
+
+## Consul Certificate
+ `consul_cert.pem` is a self-signed certificate used for configuring Consul with TLS. The private key for the certificate is `consul_key.pem`. The same certificate is used by CTS as the CA authority for communicating with Consul on a TLS connection.
+
+The DNS name for the certificate is `example.com` and the IP address is `127.0.0.1`. The certificate has a key usage restriction where it cannot be used for client authentication.
+
+<details>
+<summary>Example Consul Configuration Snippet</summary>
+```
```

Comment:
This nice markdown formatting isn't rendering as expected for me [here](https://github.com/hashicorp/consul-terraform-sync/blob/3f9b1adef0844c2970be48b0068ad009e9792674/testutils/certs/README.md)

Screenshot:
<img width="981" alt="Screen Shot 2021-11-09 at 3 09 52 PM" src="https://user-images.githubusercontent.com/6819378/140997253-1a7d50ca-5a90-4539-ba68-7eeba52a1df9.png">

Reply:
Thanks for checking this! Looks like it needed a newline between the summary and the codeblock.